src/Features/Onboarding.php

```diff
@@ -583,13 +583,9 @@ public function component_settings( $settings ) {
 			'profile'    => $profile,
 		);
 
-		// Only fetch if the onboarding wizard is incomplete.
-		if ( self::should_show_profiler() ) {
-			$settings['onboarding']['activeTheme'] = get_option( 'stylesheet' );
-		}
-
 		// Only fetch if the onboarding wizard OR the task list is incomplete.
 		if ( self::should_show_profiler() || self::should_show_tasks() ) {
```

Comment:
Maybe as a temporary workaround, we also check if the path is the profiler and load this data as well.

`isset( $_GET['path'] ) && '/profiler' === $_GET['path']`